crates/astria-conductor/src/executor/mod.rs

```diff
@@ -214,14 +209,6 @@ impl Executor {
     /// execution block hash.
     /// if there are no relevant transactions in the SequencerBlock, it returns None.
     async fn execute_block(&mut self, block: SequencerBlockSubset) -> Result<Option<Block>> {
```

Comment:
change this to just return `Result<Block>` and update the comment above

Reply:
ack, this signature can now be simplified

Reply:
updated!